README.md

```diff
@@ -40,8 +40,8 @@ Owner is a person who participates in administrative activities of the
 
 #### Rights ####
 
-- Owners have full owner access to [@morganey-lang].
-- Owners can use any technical measures available to them to regulate
+- Owners have owner access to [@morganey-lang].
```

Comment:
Re: both. Is there full and non-full admin/owner access in GitHub model of security?